src/java/org/apache/cassandra/service/snapshot/SnapshotManifest.java

```diff
@@ -26,7 +26,7 @@
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.apache.cassandra.config.Duration;
+import org.apache.cassandra.config.CassandraDuration;
```

Comment:
super nit: While we're here, we could move this import down to where it belongs.